src/components/containers/popover/Popover.js

```diff
@@ -132,12 +133,26 @@ const PopoverTitle = styled.div`
   background-color: ${colors.accent};
   color: ${colors.white};
   padding: 8px 14px;
+  flex-grow: 1;
 `;
 
 const PopoverBodyContent = styled.div`
   padding: 9px 14px;
 `;
 
+const DismissPopover = styled(DismissButton)`
+  background-color: ${props => (props.hasTitle ? colors.accent : 'none')};
```

Comment:
It looks like there are some discrepancies between the heights of the title and the close button that results in the title section background looking like this. I think the easiest fix would be to apply the background color to the `PopoverHeader` instead.

![screen shot 2017-06-28 at 12 39 17 pm](https://user-images.githubusercontent.com/3229671/27654234-d66890ba-5bfe-11e7-8ce8-a65a6e6903a6.png)